e2e/serial/swap/2_swapFlow2.test.ts

```diff
@@ -131,14 +131,6 @@ it('should be able to go to review a unlock and swap', async () => {
     id: `${SWAP_VARIABLES.USDC_MAINNET_ID}-token-to-sell-row`,
     driver,
   });
-  await findElementByTestIdAndClick({
-    id: 'token-to-buy-search-token-input',
-    driver,
-  });
-  await findElementByTestIdAndClick({
-    id: `${SWAP_VARIABLES.ETH_MAINNET_ID}-favorites-token-to-buy-row`,
-    driver,
-  });
```

Comment:
Removed this since we select native asset as output when non native asset is selected as input.